src/cdk/dialog/dialog.md

```diff
@@ -1,6 +1,16 @@
 The `Dialog` service can be used to open unstyled modal dialogs and to build your own dialog
 services.
 
+### Initial setup
+The CDK dialogs depend on a small set of structural styles to work correctly. If you're using
+Angular Material, these styles have been included together with the theme, otherwise if you're
+using the CDK on its own, you'll have to include the styles yourself. You can do so by importing
+the prebuilt styles in your global stylesheet:
```

Comment:
Might be worth mentioning that one can also get these styles by including the `cdk.overlay` mixin.

Reply:
@crisbeto I've updated the instructions to include an alternative method using the cdk.overlay mixin in Sass.